app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -144,6 +144,10 @@ def group_members
         .order('COUNT(users.id) DESC')
         .group('users.id')
         .select('users.*, COUNT(users.id) as "assignments_count"')
+
+      members = members.where("users.name like :filter OR users.username like :filter", filter: "%#{params[:filter]}%") if params[:filter]
```

Comment:
Interpolating things into SQL strings like this is potentially risky (because someone could inject malicious SQL). I think we should be able to do `members.filter_by_username()` in the same way that the [groups members list does it](https://github.com/discourse/discourse/blob/master/app/controllers/groups_controller.rb#L231)